app/src/main/res/layout/item_row.xml

```diff
@@ -41,6 +41,16 @@
                     android:textSize="24sp"
                     android:transitionName="profileName" />
 
+            <TextView
+                    android:id="@+id/timeStamp"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_below="@+id/ProfileName"
+                    android:layout_alignStart="@+id/ProfileName"
+                    android:layout_marginTop="2dp"
+                    android:text="TimeStamp"
```

Comment:
Let's not pass a default value of timeStamp textview.
Either change it to tools:text="TimeStamp" or remove this line.

Reply:
@rob729 okk